Command/RemoveThumbsCommand.php

```diff
@@ -147,11 +140,12 @@ public function execute(InputInterface $input, OutputInterface $output)
     public function getProvider()
     {
         $providerName = $this->input->getArgument('providerName');
+
         if (null === $providerName) {
             $providers = array_keys($this->getMediaPool()->getProviders());
-            $dialog = $this->getHelperSet()->get('dialog');
-            $providerKey = $dialog->select($this->output, 'Please select the provider', $providers);
-            $providerName = $providers[$providerKey];
+
+            $question = new ChoiceQuestion('Please select the provider', $providers);
```

Comment:
This variable seems to be use only one time.

It should not be initialized.

cc @greg0ire